### PR TITLE
Update Clio.com

### PR DESCRIPTION
**Pull request by yegct**
Clio changed from goclio.com to clio.com and deprecated email 2FA in favour of Google Auth. Information as provided in the existing documentation link.

Clio is legal practice management software, so moved from 'other' to 'legal'.

### CODE REVIEW

File: _data/legal.yml [hunk 1]
```diff
@@ -1,4 +1,11 @@
 websites:
```
Comment (cliochris) [inline _data/legal.yml]: Moved from other.yml to legal.yml

File: _data/legal.yml [hunk 2]
```diff
@@ -1,4 +1,11 @@
...
+    - name: Clio
+      url: https://www.clio.com
```
Comment (cliochris) [inline _data/legal.yml]: Changed from www.goclio.com to www.clio.com.

File: _data/legal.yml [hunk 3]
```diff
@@ -1,4 +1,11 @@
...
+      img: clio.png
+      tfa: Yes
+      software: Yes
+      doc: https://support.goclio.com/hc/en-us/articles/203184210-Two-Factor-Authentication
```
Comment (cliochris) [inline _data/legal.yml]: Unchanged.

File: _data/legal.yml [hunk 4]
```diff
@@ -1,4 +1,11 @@
 websites:
+    - name: Clio
+      url: https://www.clio.com
+      img: clio.png
+      tfa: Yes
+      software: Yes
```
Comment (cliochris) [inline _data/legal.yml]: Removed email, added software. 2FA is provided through Google Authenticator (actually, anything that supports T-OTP).

**Comment by mxxcon** [comment]
I deleted your in-code comments to make diff easier. Your PR comments describe all the changes sufficiently clearly. 👍

**Review by mxxcon — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Comment by Carlgo11** [comment]
Thanks for the PR! Even though your PR has been merged, it might take some time before it shows up on the website. For more information, check out our [wiki](https://github.com/2factorauth/twofactorauth/wiki/FAQ-Merge).

**Pull request merged**